templates/default/engage-nginx-proxy-conf.erb

```diff
@@ -1,6 +1,7 @@
 server {
   listen 80 default_server;
   listen [::]:80 default_server ipv6only=on;
+  directio 1M;
```

Comment:
And 1M was based on a `find` command I ran that found no JPG thumbnails were larger than that on the prod FS.